### DISCUSSION TITLE
Anyone maintaining feed gen repo?

### DISCUSSION THREAD

**Discussion by ebwinters** [bluesky-social/atproto#2246] [General Discussion]
While learning about feed gen a few months back I found an issue with the way the template was set up from a code POV. I submitted a PR last December for a change to the feed generator template, but haven't seen any merges in 3 months and no PRs have gotten reviewed either, including mine. Just wondering if I should move on and assume feed generator is not maintained, or spend more time understanding it- thanks

**Comment by bnewbold** [comment]
Our main motivation is to have a demonstration of how a feed generator might work, not necessarily have it be a general template or be very full-featured. If there are bugs, or if we make changes/additions to how the feed generator "API" works, I think we will definitely update the repo to match that.

It would be great if there were multiple examples or templates in the community, which take different coding approaches (techniques, languages, frameworks) and demonstrate design trade-offs.

**Reply by ebwinters** [reply]
Definitely understand the purpose of that repo and agree more examples would be better. My point is, for someone new to the atproto space trying to look at code+docs, the actual template (what's-alf) confused me because of where the filtering was done vs where the fire hose feed was input. I created a PR based on my understanding, but got no review for a few months now, so I was wondering if the repo is active or in a 'change only when needed' state- like you mentioned when a breaking change comes. It seemed like an easier place to start and try to make a first contribution (rather than the protocol layer), so that is where this discussion is stemming from
